Tasks/between.js

```diff
@@ -1,21 +1,13 @@
 // Extract substring between prefix and suffix
 
-getvaluebetween = (str, p, s) => {
-  i = str.indexOf(p);
-  if (i === -1) return '';
-  else {
-    k = i + p.length;
-    str = str.substring(k);
-    if (s) {
-      i = str.indexOf(s);
-      if (i === -1) {
-        return '';
-      } else {
-        str = str.substring(0, i);
-      }
-    }
+const getValueBetween = (str, p, s) => {
+  const prefixStartIndex = str.indexOf(p);
+  const prefixEndIndex = prefixStartIndex + p.length;
+  const suffixStartIndex = str.indexOf(s);
+
+  if ([prefixStartIndex, suffixStartIndex].includes(-1)) {
```

Comment:
Using or `||` operator is better then creating array and call `includes` method (this will iterate array)
```suggestion
  if (prefixStartIndex === -1 || suffixStartIndex === -1) {
```